service/lib/dinstaller/manager.rb

```diff
@@ -89,7 +90,6 @@ def install_phase
       start_progress(8)
 
       progress.step("Reading software repositories") do
```

Comment:
I woud say this step is not true anymore.

Reply:
right, it should be already read.